index.html

```diff
@@ -8,75 +8,89 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <header class="portfolio">
+  <header class="portfolio">
 		<a href="#">S</a>
-			<nav class="nav-menu">
-				<a href="#"><img src="Assets/Icon-Menu.png" alt=""></a>
+			<nav class="nav-list">
+        <ul class="nav-menu">
+          <li><a href="#recent">Portfolio</a></li>
+          <li><a href="#about-me">About</a></li>
+          <li><a href="#form-footer">Contact</a></li>
+          <li><a href="#"><img src="Assets/Letter.png" alt=""></a></li>
+      </ul>
+				<a class="icon-menu" href="#"><img src="Assets/Icon-Menu.png" alt=""></a>
 			</nav>
-	</header>
-
-		<div class="personal-info">
+	</header> 
+  
+  <section class="personal-info">
+    <div class="brief-intro">
 			<h1>Hey There. <br> I am Shakur</h1>
 			<h3>I am a Software Developer</h3>
 			<p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
 				don’t hestiate to contact me.</p>
+      </div>
 				<ul class="links">
 					<li class="lists"><a href="#"> <img src="Assets/github.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/Linkedin.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/Vector.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/twitter.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/medium.png" alt=""></a></li>
 				</ul>
-		 
-		</div>
-		  <h2 class="recent">My Recent Works</h2>
-		<div class="works">
-			<div class="multi">
-				<h2 class="multi-post">Multi-Post Stories</h2>
-			<p>A daily selection of privately personalized reads;
-				 no accounts or sign-ups required. has been the 
-				 industry's standard dummy text ever since the 1500s, 
-				 when an unknown printer took a standard dummy text.</p>
-			<ul class="buttons">
-				<li class="btn-skills">css</li>
-				<li class="btn-skills">html</li>
-				<li class="btn-skills">bootstrap</li>
-				<li class="btn-skills">Ruby</li>
-			</ul>
-			<div class="btn-skls">
-        <button class="btn-skills-project">See Project</button>
-      </div>				
-      
-			</div>
-		</div>
+  </section>
 
-		<div class="data">
+  <section class="recent-profession">
+          <h2 class="recent-header">My Recent Works</h2>          

```

Comment:
kindly align the text here to the left 

#### Original design
![wrong2](https://user-images.githubusercontent.com/58376020/122623402-38b82580-d09c-11eb-933b-39b2c7c57a62.png)

##### Your Project
![right2](https://user-images.githubusercontent.com/58376020/122623409-42418d80-d09c-11eb-9a8b-92a95484eacd.png)